go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/pingcap/errors v0.11.0
 	github.com/pingcap/goleveldb v0.0.0-20171020122428-b9ff6c35079e
 	github.com/pingcap/kvproto v0.0.0-20181105061835-1b5d69cd1d26
-	github.com/pingcap/parser v0.0.0-20181211162423-79b42ecfaf70
+	github.com/pingcap/parser v0.0.0-20181214132045-732efe993f70
```

Comment:
```
➜  tidb git:(master) ✗ GO111MODULE=on go get -u -v github.com/pingcap/parser@732efe993f70da99fdc18acb380737be33f2333a
➜  tidb git:(master) ✗ GO111MODULE=on go mod tidy
```

Then `make dev` on this branch, make sure the all test passes.
If the CI still fail, I suspect that something is wrong with the downloaded mod cache

Reply:
see pingcap/tidb#8694 please, i only changed a proposal, but the ci also fail

Reply:
`make dev` success? so it must be the later case @leoppro